microsoft-edge/webview2/roadmap.md

```diff
@@ -56,3 +56,6 @@ You can access WebView2 controls in your UWP applications using [Windows UI Libr
 ### Windows UI Library 3
 
 You can access WebView2 controls in your applications using [Windows UI Library 3 (WinUI 3)](/uwp/toolkits/winui3/index) with the Windows App SDK.  This has reached GA.
+
+### Xbox, HoloLens, and XAML limitations
```

Comment:
Please add an empty line between this h3 header and the paragraph below it.

Reply:
fixed